packages/components/src/internal/actions.ts

```diff
@@ -414,34 +414,34 @@ export interface SelectResponse {
     count: number;
 }
 
-export function clearSelected(
-    key: string,
-    schemaQuery?: SchemaQuery,
-    filterArray?: Filter.IFilter[],
-    containerPath?: string,
-    queryParameters?: Record<string, any>,
-    containerFilter?: Query.ContainerFilter
-): Promise<SelectResponse> {
+export type ClearSelectedOptions = {
+    containerFilter?: Query.ContainerFilter;
+    containerPath?: string;
+    filters?: Filter.IFilter[];
+    queryParameters?: Record<string, any>;
+    schemaQuery?: SchemaQuery;
+    selectionKey: string;
+};
+
+export function clearSelected(options: ClearSelectedOptions): Promise<SelectResponse> {
```

Comment:
Additionally, I've added `clearSelected` and `replaceSelected` to the query API wrapper.